library-extensions/src/main/java/com/mikepenz/fastadapter_extensions/drag/SimpleDragCallback.java

```diff
@@ -63,6 +66,17 @@ public boolean isLongPressDragEnabled() {
 
     @Override
     public boolean onMove(RecyclerView recyclerView, RecyclerView.ViewHolder viewHolder, RecyclerView.ViewHolder target) {
+
+        // remember the from/to positions
+        if (target.itemView.getTag() instanceof IDraggable) {
+            if (((IDraggable) target.itemView.getTag()).isDraggable()) {
+                if (mFrom == null) {
```

Comment:
Are you sure this is needed? Because when you start moving a second time from another positon the mFrom is wrong?